File: 14.c
#include <stdio.h>

int main()
{
    int num;
    printf("Enter a number: ");
    scanf("%d", &num);
    if (num % 3 == 0 && num % 5 == 0) 
    {
        printf("divisible\n");
    } 
    else 
    {
        printf("not divisible\n");
    }
    return 0;
}

File: 14.js
const readline = require("readline").createInterface({
    input: process.stdin,
    output: process.stdout
});
readline.question("Enter a number: ", input => {
    const num = parseInt(input);
    if (num % 3 === 0 && num % 5 === 0) {
        console.log("divisible");
    } else {
        console.log("not divisible");
    }
    readline.close();
});
